tests/functionals_tests/test_moment_coefficient.py

```diff
@@ -16,7 +16,7 @@ def test(self):
 
         comp = MomentCoefficient(surfaces=surfaces)
 
-        run_test(self, comp)
+        run_test(self, comp, complex_flag=True, method='cs')
```

Comment:
It sounds like you found a derivative error. Were these tests failing before? Or were they passing with bugs and not caught because they were using finite difference instead of complex step? Can you verify that they were failing without your changes if you switched to complex step?

Reply:
Tests pass with fd too. I was just using cs while debugging and left it that way.